src/github/handlers/repository-dispatch.ts

```diff
@@ -46,6 +50,9 @@ export async function repositoryDispatch(context: GitHubContext<"repository_disp
     await context.eventHandler.pluginChainState.put(pluginOutput.state_id, state);
     return;
   }
+  if (!isGithubPlugin(nextPlugin.plugin)) {
+    throw new Error("Trying to call a non-github plugin.");
```

Comment:
if previous plugin is a github plugin, then next plugin might be a worker plugin so you should call `dispatchWorker` or `dispatchWorkflow` depending on what the next plugin is